src/autocomplete/CommandProvider.js

```diff
@@ -0,0 +1,79 @@
+import AutocompleteProvider from './AutocompleteProvider';
+import Q from 'q';
+import Fuse from 'fuse.js';
+
+const COMMANDS = [
```

Comment:
In the longer term, we should try to get this information from the definitions of the commands in src/SlashCommands.js rather than duplicating the usage text here